webapps/plf-sites-extension/src/main/webapp/WEB-INF/conf/sites/portal/global/pages.xml

```diff
@@ -645,85 +645,85 @@
             </portlet-application>
           </container>
           <container
-            id="contributionsContainer"
+            id="TopChallengersContainer"
             template="system:/groovy/portal/webui/container/UIVColContainer.gtmpl"
             cssClass="col col-sm-12 col-md-6 col-lg-4">
             <portlet-application>
               <portlet>
                 <application-ref>gamification-portlets</application-ref>
-                <portlet-ref>myContributions</portlet-ref>
+                <portlet-ref>topChallengers</portlet-ref>
               </portlet>
-              <title>My Contributions</title>
+              <title>Top Challengers</title>
               <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
               <show-info-bar>false</show-info-bar>
             </portlet-application>
           </container>
           <container
-            id="ReputationContainer"
+            id="ChanllengesOverviewContainer"
```

Comment:
typo here

Reply:
Yes, thanks, it was defined in develop, [see it here](https://github.com/Meeds-io/meeds/blob/develop/webapps/plf-sites-extension/src/main/webapp/WEB-INF/conf/sites/portal/global/pages.xml#L704)

Knowing that there is no specific CSS to this id, I can manage to change it without affecting the UI. Fixed.

```suggestion
            id="ChallengesOverviewContainer"
```